pkg/linear-pools/contracts/tetu-linear-pool/TetuLinearPool.sol

```diff
@@ -63,11 +60,11 @@ contract TetuLinearPool is LinearPool, Version, TetuShareValueHelper {
         ITetuSmartVault tokenVault = ITetuSmartVault(address(args.wrappedToken));
 
         _require(address(args.mainToken) == tokenVault.underlying(), Errors.TOKENS_MISMATCH);
-
-        _mainToken = args.mainToken;
         _wrappedToken = args.wrappedToken;
+    }
 
-        _rateScaleFactor = 10**(SafeMath.sub(18, ERC20(tokenVault.underlying()).decimals()));
+    function _getWrappedTokenRate() internal view override returns (uint256) {
```

Comment:
I changed for alphabetical reasons, and also because _getWrappedTokenRate is internal and _toAssetManagerArray is private (so private should come after internal in my head, hahahaha).

But I reverted this, to follow the pattern of other pools